registry/registry/ansible/play.yml

```diff
@@ -0,0 +1,155 @@
+---
+- hosts: registry
+  remote_user: ubuntu
+  gather_facts: False
+  vars_files:
+    - vars.yml
+  # hack for ubuntu 16.04
```

Comment:
There is no hack. There was a hack when the image was not behaving but they sorted that out. I will remove the comment.